src/main/java/seedu/dictionote/model/NoteBook.java

```diff
@@ -0,0 +1,96 @@
+package seedu.dictionote.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.dictionote.model.note.Note;
+import seedu.dictionote.model.note.UniqueNoteList;
+
+/**
+ * Wraps all data at the notebook level
+ * Duplicates are not allowed (by .isSamePerson comparison)
+ */
+public class NoteBook implements ReadOnlyNoteBook {
+    private final UniqueNoteList notes;
+
+    {
+        notes = new UniqueNoteList();
+    }
+
+    public NoteBook() {}
```

Comment:
Should the code block in lines 18-20 be written inside the constructor in line 22?

Reply:
I copy this design from the addressbook code,

Reply:
We copy this from the original implementation of AddressBook

Reply:
Oh I see, sorry for the misunderstanding.